MODERNIZATION_PLAN.md

```
@@ -106,87 +127,65 @@ Angular 21 migration completed in v0.11.0 (fresh rewrite, not based on earlier v
+|  +-------------+                                                  |
++------------------------------------------------------------------+
 ```
 
 ---
 
 ## Known Limitations
 
-### 1. scanfs Binary Compatibility (Fixed in v0.9.4)
-- PyInstaller binary built on Debian Bullseye (glibc 2.31) for broad compatibility
-- Supports Linux systems from 2021+
+### 1. scanfs Compatibility
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)